client/state/themes/actions.js

```diff
@@ -272,6 +223,15 @@ export function requestThemes( siteId, query = {} ) {
 	};
 }
 
+export function requestThemeFailure( siteId, themeId, error ) {
```

Comment:
Will fix in #9809. I will correct name. I will go with `themeRequestFailure` if you think it is OK. Still I would like to have it separate as it is also used in `ThemeSheet` test.